autoscheduler/frontend/src/package.json

```diff
@@ -48,7 +46,6 @@
     "@reach/router": "^1.2.1",
     "@types/reach__router": "^1.2.6",
     "@types/react-redux": "^7.1.5",
-    "prop-types": "^15.7.2",
```

Comment:
Do we not have a need for `prop-types` anymore? Is it just needed by router as a dependency? I know you removed `react, react-dom`, and `prop-types` from `devDependencies` to silence the `included as both a dev and production dependency` warnings, but this is removing `prop-types` from both.

Reply:
Yeah, it was intentional, since we aren't really using prop-types as a top-level dependency in either development or production.